packages/ui/src/mocks/resolvers.ts

```diff
@@ -39,42 +44,63 @@ export const getMemberResolver = (obj: any, args: any, context: any, info: any)
   return mirageGraphQLFieldResolver(obj, resolverArgs, context, info)
 }
 
-type SortFn = (a: MockMember, b: MockMember) => number
-const getSortFn = (orderBy?: MembershipOrderByInput): void | SortFn => {
+interface SortableQueryResult {
+  id: string
+  handle: string
+}
+const sort = <T extends SortableQueryResult>(members: T[], orderBy?: Maybe<MembershipOrderByInput>): T[] => {
   const { EntryAsc, EntryDesc, HandleAsc, HandleDesc } = MembershipOrderByInput
 
   const authorizedKeys = [EntryAsc, EntryDesc, HandleAsc, HandleDesc]
   if (!orderBy || !authorizedKeys.includes(orderBy)) {
-    return
+    return members
   }
 
   const [key, direction] = orderBy.toLowerCase().split('_')
   const membersKey = (key === 'entry' ? 'id' : key) as 'id' | 'handle'
   const fact = direction === 'desc' ? 1 : -1
+  const sortFn = (a: T, b: T) => fact * (b[membersKey] ?? '').localeCompare(a[membersKey] ?? '')
```

Comment:
Try to avoid abbreviations. I'd go with something like:

```suggestion
  const sortByKey = (a: T, b: T) => fact * (b[membersKey] ?? '').localeCompare(a[membersKey] ?? '')
```

Reply:
Or even:  
 

```typescript
const sortByKey = (key: key of Member) => {
  return (a: T, b: T) => fact * (b[key] ?? '').localeCompare(a[key] ?? ''
}
```

then use it:

```typescript
members.sort(sortByKey(membersKey))
```

However, this is a code used for mocks. So, we don't need to spend much time on it ;) Renaming it to `sortByKey` is sufficient.